simpleui/ViewController.swift

```diff
@@ -9,6 +9,137 @@
 import UIKit
 
 class ViewController: UIViewController {
+    
+    //counter
+    var count: Int = 0
+    @IBOutlet var counter_label: UILabel!
```

Comment:
In Swift ist camelCase die Konvention für Variablennamen, schreib also lieber `counterLabel`